### PR TITLE
Added FormAPI::getInstance();

### PR DESCRIPTION
**Pull request by uramnoil**

### CODE REVIEW

**Comment by SOF3** [comment]
What prevents you from using `$server->getPluginManager()->getPlugin("FormAPI")`?

**Comment by Ifera** [comment]
Laziness

**Comment by dktapps** [comment]
lol no generics

**Comment by SOF3** [comment]
Then why not `return Server::getInstance()->getPluginManager()->getPlugin("FormAPI")` instead of maintaining your own singleton?

File: src/jojoe77777/FormAPI/FormAPI.php [hunk 1]
```diff
@@ -16,11 +16,21 @@ class FormAPI extends PluginBase implements Listener {
     public $formCount = 0;
     /** @var array */
     public $forms = [];
+    /** @var FromAPI */
```
Comment (SOF3) [inline src/jojoe77777/FormAPI/FormAPI.php:19]: typo here

**Review by SOF3 — CHANGES_REQUESTED** [review]

**Comment by uramnoil** [comment]
Oh my god, I totally forgot 'Server::getInstance()'...
Please forgive my mistake:(

**Comment by uramnoil** [comment]
That's certainly laziness lol

**Comment by jojoe77777** [comment]
Unneeded as stated above

**Pull request closed without merging**